src/matching/Matching_generic.ml

```diff
@@ -820,6 +820,36 @@ let adjust_info_remove_enclosing_quotes (s, info) =
           (* return original token ... better than failwith? *)
           (s, info))
 
+let clean_php_docstring raw =
```

Comment:
Nit: `remove_php_doc_string_delimiters` might be a better name?

Reply:
I agree. Renamed.